File: Dart/1009.dart
import 'dart:io';

void main() {
  String name = stdin.readLineSync();
  double salary = double.parse(stdin.readLineSync());
  double sales = double.parse(stdin.readLineSync());
  if (sales < 0) {
    print(salary);
  } else {
    double bonus = sales * 0.15;
    double total = salary + bonus;
    print("TOTAL = R\$ " + total.toStringAsFixed(2));
  }
}

File: PHP/1009.php
<?php
$name = rtrim(fgets(STDIN));
$salary = rtrim(fgets(STDIN));
$sales = rtrim(fgets(STDIN));

$percentage = $sales * 0.15;
$total = $salary + $percentage;

echo "TOTAL = R$ " . number_format($total, 2, ".", "") . PHP_EOL;
